src/reactviews/pages/QueryResult/table/utils.ts

```diff
@@ -75,3 +75,18 @@ export function tryCombineSelections(selections: ISlickRange[]): ISlickRange[] {
     }
     return [unifiedSelection];
 }
+
+export function deepClone<T>(obj: T): T {
```

Comment:
This seems like it could be useful in other react views; let's put it in `reactviews/common/utils.ts`